cli/command/formatter/node.go

```diff
@@ -155,3 +157,29 @@ func (c *nodeContext) Label(name string) string {
 	}
 	return c.v.Labels[name]
 }
+
+func (c *nodeContext) Region() string {
+	c.AddHeader(nodeRegionHeader)
+	if c.v.Labels == nil {
+		return ""
+	}
+
+	if val, ok := c.v.Labels["iaas/region"]; ok {
+		return val
+	}
+
+	return ""
+}
+
+func (c *nodeContext) FailureDomain() string {
+	c.AddHeader(nodeFailureDomainHeader)
+	if c.v.Labels == nil {
+		return ""
+	}
+
+	if val, ok := c.v.Labels["iaas/failure-domain"]; ok {
+		return val
+	}
+
+	return ""
+}
```

Comment:
Would be great to have a test similar to `TestClusterHealthWrite` that would test the output of node ls, including the above region and failure domain labels.